src/Native/CMakeLists.txt

```diff
@@ -126,18 +126,14 @@ if(CLR_CMAKE_PLATFORM_ARCH_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
   add_definitions(-D_AMD64_)
   add_definitions(-DAMD64)
-  add_definitions(-DTARGET_X64)
-  add_definitions(-DTARGET_AMD64)
   add_definitions(-D_WIN64=1)
 elseif(CLR_CMAKE_PLATFORM_ARCH_I386)
   add_definitions(-D_TARGET_X86_=1)
   add_definitions(-D_X86_)
-  add_definitions(-DTARGET_X86)
   add_definitions(-D_WIN32=1)
 elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
   add_definitions(-D_TARGET_ARM_=1)
   add_definitions(-D_ARM_)
-  add_definitions(-DTARGET_ARM)
   add_definitions(-D_WIN32=1)
```

Comment:
Doing the same cleanup for _WIN64 would be nice as well (it should be only defined when building on 64-bit Windows), but it would require find`n`replace of most _WIN64 uses with BIT64.